composer/workflows/dataflowtemplateoperator_create_dataset_and_table_helper.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is a helper function to the Dataflow Template Operator Tutorial.
+# It helps the user set up a BigQuery dataset and table that is needed
+# for the tutorial.
+
+# [START composer_dataflow_dataset_table_creation]
+
+from google.cloud import bigquery
+
+# [END composer_dataflow_dataset_table_creation]
+
+
+def create_dataset(project, dataset_uuid):
+    # [START composer_dataflow_dataset_table_creation]
+
+    # Make sure to set your Google application credentials beforehand. See instructions here: https://cloud.google.com/bigquery/docs/quickstarts/quickstart-client-libraries
+   
+    # Construct a BigQuery client object.
+    client = bigquery.Client()
+
+    # TODO(developer): Set project to your GCP Project ID.
+    # project = "your-client-project"
+
+    dataset_id = f"{project}.average_weather"
+
+    # [END composer_dataflow_dataset_table_creation]
+
+    dataset_id = f"{project}.{dataset_uuid}"
```

Comment:
A small suggestion.

This is overriding the dataset_id with the provided dataset_uuid. This might confuse the reader of the code, so can you add some comments explaining why we're doing this?